core/src/test/java/io/fabric8/maven/core/util/AsciiDocParserTest.java

```diff
@@ -0,0 +1,99 @@
+package io.fabric8.maven.core.util;
+
+import java.io.ByteArrayInputStream;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+public class AsciiDocParserTest {
+
+    private static final String VALID_TABLE = "cols=2*,options=\"header\"]" + System.lineSeparator()
+        + "|===" + System.lineSeparator()
+        + "|Kind" + System.lineSeparator()
+        + "|Filename" + System.lineSeparator()
+        + System.lineSeparator()
+        + "|cm" + System.lineSeparator()
+        + "a|ConfigMap" + System.lineSeparator()
+        + System.lineSeparator()
+        + "|cronjob" + System.lineSeparator()
+        + "|CronJob" + System.lineSeparator()
+        + "|===";
+
+    private static final String NONE_END_VALID_TABLE = "cols=2*,options=\"header\"]" + System.lineSeparator()
+        + "|===" + System.lineSeparator()
+        + "|Kind" + System.lineSeparator()
```

Comment:
For consistencies sake, I would change this header, too (also it doesnt really matter in the test)

Reply:
obsolete as we decided to go for Kind, Filename.